cards/en/bw11.json

```diff
@@ -3466,6 +3466,7 @@
     "number": "58",
     "artist": "Sumiyoshi Kizuki",
     "rarity": "Rare",
+    "flavorText": "Its cry sound like an incantation. It is said the cry may rarely be imbued with happiness-giving power.",
```

Comment:
Small typo here, should be `sounds` instead of `sound`. "Its cry sounds like...."

Reply:
Fixed it, thanks